framework/Objects/Meta_Fields.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+namespace JustCoded\WP\Framework\Objects;
+
+/**
+ * Meta_Fields is the base class for acquiring data from custom fields.
+ */
+
+class Meta_Fields {
```

Comment:
class should be abstract, because you can use it correctly. I think it can be called as "Meta".

Reply:
renamed